src/components/link/link.test.js

```diff
@@ -0,0 +1,178 @@
+import * as React from "react";
+import Link from "./link.component";
+
+import {
+  icon,
+  tooltipPreview,
+  link,
+  getDataElementByValue,
+  body,
+} from "../../../cypress/locators";
+import { skipLink } from "../../../cypress/locators/link/index";
+import { keyCode } from "../../../cypress/support/helper";
+import CypressMountWithProviders from "../../../cypress/support/component-helper/cypress-mount";
+
+const testData = ["mp150ú¿¡üßä", "!@#$%^*()_+-=~[];:.,?{}&\"'<>"];
+const testCypress = "test-cypress";
+
+const LinkComponent = ({ ...props }) => {
+  return (
+    <div
+      style={{
+        margin: "80px",
+      }}
+    >
+      <Link href="#foo" target="_blank" rel="noreferrer noopener" {...props}>
+        This is a link
+      </Link>
+    </div>
+  );
+};
+
+context("Test for Link component", () => {
+  describe.only("Check props for Link component", () => {
```

Comment:
Do we need ".only" here? Also "Check" should be "check"

Reply:
good spot